### PR TITLE
New Package: mac-fdisk

### PR DESCRIPTION
**Pull request by stenstorp**
Required for editing Apple Partition Map drives.
No longer hosted by the original creator so I used the source used by Debian.
Patches are from Debian and Gentoo.

### CODE REVIEW

**Comment by jnbr** [comment]
at this point it would probably make sense create a fork with all these patches applied

File: srcpkgs/mac-fdisk/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'mac-fdisk'
+pkgname=mac-fdisk
+version=0.1
+revision=1
+only_for_archs="i686 i686-musl x86_64 x86_64-musl ppc ppc-musl ppc64-musl ppc64le ppc64le-musl"
```
Comment (jnbr) [inline srcpkgs/mac-fdisk/template]: is there a reason why this can't be build for the remaining arches?

Comment (stenstorp) [reply srcpkgs/mac-fdisk/template]: Yes, it fails to build on arm and. Would it be better to mark it as broken on arm?

Comment (jnbr) [reply srcpkgs/mac-fdisk/template]: yes

File: srcpkgs/mac-fdisk/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'mac-fdisk'
+pkgname=mac-fdisk
+version=0.1
+revision=1
+wrksrc=${pkgname}-${version}.orig
+build_style=gnu-makefile
+short_desc="Format and edit Apple Partition Map drives"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="ftp://ftp.mklinux.apple.com/pub/Other_Tools/"
```
Comment (maxice8) [inline srcpkgs/mac-fdisk/template]: ftp homepage ?

Comment (stenstorp) [reply srcpkgs/mac-fdisk/template]: That's what the homepage is on the debian and gentoo packages. It's a pretty old program and the homepage went down ages ago. You want me to set it to the debian package page?
I should also replace only_for_arches with the new thing.

**Comment by maxice8** [comment]
On Tue, Feb 19, 2019 at 02:05:48PM -0800, stenstorp wrote:
> stenstorp commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,28 @@
> +# Template file for 'mac-fdisk'
> +pkgname=mac-fdisk
> +version=0.1
> +revision=1
> +wrksrc=${pkgname}-${version}.orig
> +build_style=gnu-makefile
> +short_desc="Format and edit Apple Partition Map drives"
> +maintainer="Thomas Batten <stenstorpmc@gmail.com>"
> +license="GPL-2.0-or-later"
> +homepage="ftp://ftp.mklinux.apple.com/pub/Other_Tools/"
> 
> That's what the homepage is on the debian and gentoo packages. It's a pretty old program and the homepage went down ages ago. You want me to set it to the debian package page?
> I should also replace only_for_arches with the new thing.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8165#discussion_r258250114

Set it to the debian package at least it has some use

**Pull request merged**